src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/ServiceAccountProvider.scala

```diff
@@ -71,4 +71,6 @@ abstract class ServiceAccountProvider(config: Config) {
     * @return list of users that will have reader access to a staging bucket
     */
   def listUsersStagingBucketReaders(userEmail: WorkbenchEmail)(implicit executionContext: ExecutionContext): Future[List[WorkbenchEmail]]
+
+  def getAccessToken(userEmail: WorkbenchEmail, googleProject: GoogleProject): String
```

Comment:
I think this should return a `Future[String]` and take an `implicit executionContext: ExecutionContext` similar to the other methods in this class.